src/content/app/tools/vep/views/vep-submission-results/VepSubmissionResults.tsx

```diff
@@ -0,0 +1,350 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { useVepResultsQuery } from 'src/content/app/tools/vep/state/vep-api/vepApiSlice';
+
+import { Table, ColumnHead } from 'src/shared/components/table';
+import VariantConsequence from 'src/shared/components/variant-consequence/VariantConsequence';
+import VepResultsGene from './components/vep-results-gene/VepResultsGene';
+import VepResultsLocation from './components/vep-results-location/VepResultsLocation';
+
+import type {
+  VEPResultsResponse,
+  AlternativeVariantAllele,
+  PredictedTranscriptConsequence
+} from 'src/content/app/tools/vep/types/vepResultsResponse';
+
+import styles from './VepSubmissionResults.module.css';
+
+/**
+ * TODO:
+ * - Add unique id to variants after they are requested (to use for keys)
+ * - Enable expansion/collapsing of transcripts of a given gene
+ * - Show intergenic variants as predicted consequences alongside transcript consequences
+ * - Consider pagination (should it be part of url?)
+ * - Make the table scrollable
+ */
+
+const VepSubmissionResults = () => {
+  const { currentData: vepResults } = useVepResultsQuery();
+
+  return (
+    <div className={styles.container}>
+      <div>Vep analysis</div>
```

Comment:
Shouldn't VEP be all capitalised?

Reply:
I think this was me just adding a placeholder string; but I'll capitalise it.

Reply:
✅